src/cdef.rs

```diff
@@ -20,11 +21,16 @@ use crate::src::cpu::{rav1d_get_cpu_flags, CpuFlags};
 #[cfg(feature = "asm")]
 use crate::include::common::bitdepth::BPC;
 
-pub type CdefEdgeFlags = c_uint;
-pub const CDEF_HAVE_BOTTOM: CdefEdgeFlags = 8;
-pub const CDEF_HAVE_TOP: CdefEdgeFlags = 4;
-pub const CDEF_HAVE_RIGHT: CdefEdgeFlags = 2;
-pub const CDEF_HAVE_LEFT: CdefEdgeFlags = 1;
+bitflags! {
+    #[repr(transparent)]
+    #[derive(Clone, Copy)]
+    pub struct CdefEdgeFlags: u32 {
+        const CDEF_HAVE_BOTTOM = 1 << 3;
+        const CDEF_HAVE_TOP = 1 << 2;
+        const CDEF_HAVE_RIGHT = 1 << 1;
+        const CDEF_HAVE_LEFT = 1 << 0;
+    }
+}
```

Comment:
`bitflags!` is fine and good; it's still quite close to the C conceptually.  I've converted a few like this already, but make it less of a priority since it isn't strictly necessary for safety, though it is nice to have.

Reply:
Also, unless something is in `include/` and `pub` and thus part of the C API, we can change the layout/size.  So these flags can be `u8`, and only need to be `#[repr(transparent)]` if you're using that (it's not needed for compatibility).

Reply:
this particular type is used in C FFI though in this file? I think it should be the size/repr that it has